src/core/network/polling.ts

```diff
@@ -54,10 +55,12 @@ export class Polling {
   }
 
   private async syncUpdateOffset() {
-    debug('Syncing update offset...')
-    await this.telegram
-      .callApi('getUpdates', { offset: this.offset, limit: 1 })
-      .catch(noop)
+    if (!this.skipOffsetSync) {
```

Comment:
Minor thing, I won't insist, but I'd prefer avoiding nesting by doing:
```suggestion
    if (this.skipOffsetSync) return
```